xbmc/windowing/gbm/DRMUtils.cpp

```diff
@@ -737,3 +741,23 @@ uint32_t CDRMUtils::FourCCWithoutAlpha(uint32_t fourcc)
 {
   return (fourcc & 0xFFFFFF00) | static_cast<uint32_t>('X');
 }
+
+bool CDRMUtils::CheckConnector(int connector_id)
+{
+  struct connector connectorcheck;
+  unsigned retryCnt = 7;
+
+  connectorcheck.connector = drmModeGetConnector(m_fd, connector_id);
+  while (connectorcheck.connector->connection != DRM_MODE_CONNECTED  && retryCnt > 0)
+  {
+    retryCnt--;
```

Comment:
Maybe just add a debug log for each retry (I'd be curious on my own system)

Reply:
done